client/keys.go

```diff
@@ -427,3 +447,28 @@ func (k *Key) Reseal(in *pb.SealedBytes, uOpts UnsealOpts, sOpts SealOpts) (*pb.
 func (k *Key) hasAttribute(attr tpm2.KeyProp) bool {
 	return k.pubArea.Attributes&attr != 0
 }
+
+// Cert returns the parsed certificate for the given key.
+func (k *Key) Cert() *x509.Certificate {
+	return k.cert
+}
+
+// CertRaw provides the ASN.1 DER content of the key's certificate.
+func (k *Key) CertRaw() []byte {
```

Comment:
Let's explicitly call this `CertDERBytes()` or something more specific than `Raw()`.

Reply:
Done.